api/src/test/java/org/openmrs/validator/ObsValidatorTest.java

```diff
@@ -511,4 +515,120 @@ public void validate_shouldPassValidationIfValueTextIsLessThanTheMaximumLength()
 		assertFalse(errors.hasFieldErrors("obsDatetime"));
 		assertFalse(errors.hasFieldErrors("valueText"));
 	}
+
+	/**
+	 * @see ObsValidator#validate(java.lang.Object, org.springframework.validation.Errors)
+	 */
+	@Test
+	public void validate_shouldFailValidationIfObsValueExceedsHiAbsolute() {
+		Calendar calendar = Calendar.getInstance();
```

Comment:
Do we have a corresponding test that passes when the obs value does not exceed the hi absolute?

Reply:
The test that tests "within range" satisfies this condition